webcompat/static/js/lib/bugform.js

```diff
@@ -227,6 +217,20 @@ function BugForm() {
     }
   };
 
+  this.clearStepsToReproducePlaceholder = function() {
```

Comment:
I don't see anything changed here from my comment @ https://github.com/webcompat/webcompat.com/pull/1675#discussion_r130193804.

Basically we should clear this *always* on focus. `onFormSubmit` takes care of the `details` param: `this.stepsToReproduceField.val('')`

You could set a property on the BugForm object to know if the user has clicked in here here or not, and then clear it out if they haven't. If they have, just early return. Does that make sense?